src/Common/Mailer/Configurator.php

```diff
@@ -2,64 +2,46 @@
 
 namespace Common\Mailer;
 
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use PDOException;
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
-use Frontend\Core\Engine\Model;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Common\ModulesSettings;
 
-class Configurator implements EventSubscriberInterface
+class Configurator
 {
-    private $database;
+    /**
+     * @var ModulesSettings
+     */
+    private $modulesSettings;
+
+    /**
+     * @var ContainerInterface
+     */
     private $container;
-    private $settings;
 
-    public function __construct($database, $container)
+    public function __construct(ModulesSettings $modulesSettings, ContainerInterface $container)
```

Comment:
> There MUST NOT be a hard limit on line length; the soft limit MUST be 120 characters; lines SHOULD be 80 characters or less.

-- PSR2

I think it makes sense here to keep it on one line, since it's not fully required to make this less then 80 chars. (Multiline constructors look so messy!)